app/src/main/java/org/p2p/wallet/home/ui/main/HomeInteractor.kt

```diff
@@ -44,14 +43,16 @@ class HomeInteractor(
         if (strigaSignupEnabledFeatureToggle.isFeatureEnabled) {
             strigaSignupInteractor.loadAndSaveSignupData()
             strigaUserInteractor.loadAndSaveUserStatusData()
-            if (strigaUserInteractor.isUserCreated()) {
-                loadStrigaFiatAccountDetails()
+            if (strigaUserInteractor.canLoadAccounts) {
+                loadDetailsForStrigaAccounts()
             }
         }
     }
 
-    suspend fun loadStrigaFiatAccountDetails(): Result<StrigaFiatAccountDetails> {
-        return strigaWalletInteractor.loadFiatAccountAndUserWallet()
+    suspend fun loadDetailsForStrigaAccounts(): Result<Unit> = kotlin.runCatching {
+        strigaWalletInteractor.getFiatAccountDetails()
+        strigaWalletInteractor.getCryptoAccountDetails()
+        Unit
```

Comment:
try org.p2p.core.utils.StdExtensionsKt#invokeAndForget

Reply:
return value is not used

Reply:
the `Result` actually used in the HomePresenter